conftest.py

```diff
@@ -65,6 +65,61 @@ def pytest_addoption(parser: Parser) -> None:
         default=CORPUS_PATH,
         help=f"Path to store the generated fuzzing corpus. Default: {CORPUS_PATH}.",
     )
+    parser.addoption(
+        "--model",
+        action="store",
+        default="nk3",
+        help="Select nitrokey model: nk3 or nkpk.",
+        choices=("nk3", "nkpk"),
```

Comment:
You don’t need to list the options in `help` if `choices` is set:

```
  --model={nk3,nkpk}    Select nitrokey model: nk3 or nkpk.
```

Reply:
Done